go/vt/sqlparser/ast.go

```diff
@@ -3322,6 +3348,39 @@ func (w *With) walkSubtree(visit Visit) error {
 	return nil
 }
 
+type Into struct {
+	IntoVariables Variables
```

Comment:
(minor) for consistency, you might just name this `Variables` instead of `IntoVariables` – `Into` seems redundant with the containing struct name, and the other members of the struct don't use that same prefix, so it stuck out to me.